mempool.go

```diff
@@ -2026,11 +2026,13 @@ func (mp *txMemPool) ProcessTransaction(tx *dcrutil.Tx, allowOrphan,
 
 	// Potentially accept the transaction to the memory pool.
 	var isOrphan bool
-	_, err := mp.maybeAcceptTransaction(tx, true, rateLimit)
+	missingParents, err := mp.maybeAcceptTransaction(tx, true, rateLimit)
 	if err != nil {
 		return err
 	}
 
+	isOrphan = len(missingParents) != 0
```

Comment:
There is no need to allocate a local here.  Just replace the `is !isOrphan` below with:

```Go
if len(missingParaents) == 0 {
```